docs/user-guides/exploring-perses-tooling.md

```diff
@@ -0,0 +1,99 @@
+# Exploring Perses tooling
```

Comment:
IMHO such doc is not really useful
- I think people that would want to interact with Perses without going through the UI are technical people that already know how to query APIs in general via the browser or tools like Postman.
- In the current state it's already quite easy to get to the API doc & the CLI guide 
![image](https://github.com/perses/perses/assets/7058693/16c84ebd-6e47-409b-938b-691235b2df65)

Still, it could be good to e.g have the "API" menu item to be clickable and display some kind of introduction (@nicolastakashi is this feasible?), where we could mention that the CLI allows to interact with the API as you wrote. So something quite short, a bit like what Grafana has here https://grafana.com/docs/grafana/latest/developers/http_api/.

Reply:
We probably interpret differently what you want to do there @eschabell :smile: 

@nicolastakashi made me think as well that the we could want to order the pages, for example in the user guide. 
What would be the best way to do it ? 
![image](https://github.com/perses/perses/assets/10258608/aa07df36-1a89-4319-9e4d-14bcd5a1550a)